server/chat-commands.js

```diff
@@ -621,20 +621,29 @@ const commands = {
 	'!away': true,
 	idle: 'away',
 	afk: 'away',
-	away(target, room, user) {
-		this.parse('/blockchallenges');
-		this.parse(`/blockpms ${target}`);
+	brb: 'away',
+	away(target, room, user, connection, cmd) {
+		let awayMessage = toId(cmd);
```

Comment:
I think this is confusing. Just do `let awayMessage;` and then only set it to `toId(cmd)` in the `else` when we don't have a `target`.